### PR TITLE
Updated yaml template for ingress.

### PR DESCRIPTION
**Pull request by gyliu513**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4056)

### CODE REVIEW

**Comment by gyliu513** [comment]
xref https://github.com/kubernetes/kubernetes/pull/47029

**Comment by bowei** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/4056#-:-KoOaRQotCKsoheXAjqG:bnfp4nl)*

**Comment by bowei** [comment]
:lgtm: 

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/4056#-:-KoOadR5DSfXNDb8cOAh:bg7igkv)*

**Comment by bowei** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes.github.io/4056)*

File: docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -72,13 +74,13 @@ spec:
 
 *POSTing this to the API server will have no effect if you have not configured an [Ingress controller](#ingress-controllers).*
 
-__Lines 1-4__: As with all other Kubernetes config, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.  For general information about working with config files, see [here](/docs/user-guide/deploying-applications), [here](/docs/user-guide/configuring-containers), and [here](/docs/user-guide/working-with-resources).
+__Lines 1-6__: As with all other Kubernetes config, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.  For general information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications), [configuring containers](/docs/user-guide/configuring-containers), [working with resources](/docs/user-guide/working-with-resources) and [ingress configuration rewrite](https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md#rewrite).
```
Comment (chenopis) [inline docs/concepts/services-networking/ingress.md]: deploying applications -> `/docs/tasks/run-application/run-stateless-application-deployment/`
configuring containers ->`/docs/tasks/configure-pod-container/configmap/`
managing resources -> `/docs/concepts/cluster-administration/manage-deployment/`

File: docs/concepts/services-networking/ingress.md [hunk 2]
```diff
@@ -72,13 +74,13 @@ spec:
...
 
-__Lines 8-9__: Each http rule contains the following information: A host (e.g.: foo.bar.com, defaults to * in this example), a list of paths (e.g.: /testpath) each of which has an associated backend (test:80). Both the host and path must match the content of an incoming request before the loadbalancer directs traffic to the backend.
+__Lines 10-11__: Each http rule contains the following information: A host (e.g.: foo.bar.com, defaults to * in this example), a list of paths (e.g.: /testpath) each of which has an associated backend (test:80). Both the host and path must match the content of an incoming request before the loadbalancer directs traffic to the backend.
 
-__Lines 10-12__: A backend is a service:port combination as described in the [services doc](/docs/user-guide/services). Ingress traffic is typically sent directly to the endpoints matching a backend.
+__Lines 12-14__: A backend is a service:port combination as described in the [services doc](/docs/user-guide/services). Ingress traffic is typically sent directly to the endpoints matching a backend.
```
Comment (chenopis) [inline docs/concepts/services-networking/ingress.md]: services doc -> `/docs/concepts/services-networking/service/`

File: docs/concepts/services-networking/ingress.md (hunk 1 above)
Comment (gyliu513) [inline docs/concepts/services-networking/ingress.md]: @chenopis , why want to update here? I think that current link is clear enough?

Comment (chenopis) [inline docs/concepts/services-networking/ingress.md]: The `/docs/user-guide/` directory is mostly deprecated, so the links need to be updated to their new locations.

**Pull request merged**